homeassistant/components/image_processing/facebox.py

```diff
@@ -10,32 +10,38 @@
 import requests
 import voluptuous as vol
 
-from homeassistant.const import (
-    ATTR_ENTITY_ID, ATTR_NAME)
+from homeassistant.const import ATTR_ENTITY_ID
 from homeassistant.core import split_entity_id
 import homeassistant.helpers.config_validation as cv
 from homeassistant.components.image_processing import (
     PLATFORM_SCHEMA, ImageProcessingFaceEntity, ATTR_CONFIDENCE, CONF_SOURCE,
     CONF_ENTITY_ID, CONF_NAME, DOMAIN)
-from homeassistant.const import (CONF_IP_ADDRESS, CONF_PORT)
+from homeassistant.const import (
+    CONF_IP_ADDRESS, CONF_PORT, CONF_PASSWORD, CONF_USERNAME,
+    HTTP_BAD_REQUEST, HTTP_UNAUTHORIZED)
 
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_BOUNDING_BOX = 'bounding_box'
 ATTR_CLASSIFIER = 'classifier'
 ATTR_IMAGE_ID = 'image_id'
+ATTR_ID = 'id'
 ATTR_MATCHED = 'matched'
+ATTR_NAME = 'name'
```

Comment:
We're using `ATTR_NAME` in the platform services, ie for home assistant APIs, so that we can import from const.py. For the `'name'` string that goes in calls to the facebox api, we should use another constant or no constant at all.

Please import `ATTR_NAME` from const.py and rename the constant here to something else, eg `FACEBOX_NAME`.

Reply:
The variables are getting really confusing, happy to just use `name` if preferred

Reply:
My point is that we shouldn't use the same constant when interfacing with two separate APIs. One API might change and then we might change the constant to go along with the changed API, and then we would break the usage of the other API.

Reply:
Totally agree